CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # w3c/vc-data-model-2.0-test-suite ChangeLog
 
-## 1.1.0 -
+## 1.1.0 - 01/11/2024
```

Comment:
```suggestion
## 1.1.0 - 2024-01-xx
```

Reply:
There is only ISO 8601. 😉 https://xkcd.com/1179/